AppCenterCrashes/AppCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -308,7 +308,7 @@ - (void)testProcessCrashesWithErrorAttachments {
                             }]])
       .andReturn(channelUnitMock);
   for (NSUInteger i = 0; i < invalidLogs.count; i++) {
-    OCMReject([channelUnitMock enqueueItem:invalidLogs[i]]);
+    OCMReject([channelUnitMock enqueueItem:invalidLogs[i] flags:NO]);
```

Comment:
Booleans instead of flags in many tests :blush:

Reply:
and a few in the actual implementations...fixed - good catch!